README.md

```diff
@@ -25,6 +24,8 @@ frameborder="0" allowfullscreen></iframe>
 
 ### Swift
 
+![Swift CI](https://github.com/square/workflow/workflows/Swift%20CI/badge.svg)
```

Comment:
```suggestion
![Swift CI](https://github.com/square/workflow/workflows/Swift%20CI/badge.svg?branch=master)
```

Should we pick the branch here?

Reply:
Not necessary. It defaults to the Default branch, as configured in Github, which we have set to `master`.